rpcs3/Emu/Cell/Modules/libmixer.cpp

```diff
@@ -6,6 +6,13 @@
 #include "cellAudio.h"
 #include "libmixer.h"
 
+#if __GNUC__ >= 6
+#include <math.h>
+#define FABS(X) std::fabs(X)
```

Comment:
Are you sure?
I did a quick test on gcc 4.9 and it seems to be working: https://godbolt.org/g/uPvSA6
You probably forgot to add -std=c++11 or -std=c++14 (though it's strange, because I though gcc 5 had 11 by default).